http/file_server_test.ts

```diff
@@ -198,7 +198,12 @@ Deno.test("serveDir() serves directory index with entry to the parent directory"
   assertEquals(res.status, 200);
   assertStringIncludes(
     page,
-    '<a href="../../">home</a>/<a href="../">nested-subdirs</a>/<a href=".">subdir</a>/',
+    `<a href="../../">home</a>
+        /
+          <a href="../">nested-subdirs</a>
+        /
+          <a href=".">subdir</a>
+        /`,
```

Comment:
This will render as `home / nested-subdirs / subdir /`, not `home/nested-subdirs/subdir/` as before